### DISCUSSION TITLE
Will dpgen judge whether the electronic step converges to the set precision?

### DISCUSSION THREAD

**Discussion by chunweizhu** [deepmodeling/dpgen#1154] [Q&A] [answered]
When dpgen performs vasp single-point calculation, will it check whether the calculation result reaches the accuracy set on INCAR?

Because when the ion deviation is large, the electronic step cannot converge to the set accuracy.

**Comment by AnguseZhang** [comment, marked as answer]
`DP-GEN` uses `dpdata` to read data from VASP calculations, and those frames which do not reach the expected accuracy will not be collected.

**Reply by chunweizhu** [reply]
Thanks！
